test/libsolidity/util/TestFunctionCall.cpp

```diff
@@ -141,7 +144,7 @@ string TestFunctionCall::format(
 			bytes output = m_rawBytes;
 			bool const isFailure = m_failure;
 			result = isFailure ?
-				formatFailure(_errorReporter, m_call, output, _renderResult, highlight) :
+				formatFailure(_errorReporter, m_call, output, _renderMode == RenderMode::GasResults, highlight) :
```

Comment:
Is this correct? Before your change this would always pass `true` in the fourth argument when actual results were requested. Now it's false in that case and only for `GasResults` it's `true`.

```suggestion
				formatFailure(_errorReporter, m_call, output, true, highlight) :
```

Reply:
I think It should be `_renderMode == RenderMode::ActualValuesExpectedGas`